knpu/episode1/Challenges/Extending/CallNonexistentMethodMC.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Challenges\Extending;
+
+use KnpU\ActivityRunner\Activity\MultipleChoice\AnswerBuilder;
+use KnpU\ActivityRunner\Activity\MultipleChoiceChallengeInterface;
+
+class CallNonexistentMethodMC implements MultipleChoiceChallengeInterface
+{
+    /**
+     * @return string
+     */
+    public function getQuestion()
+    {
+        return <<<EOF
+Look at these two classes:
+
+```php
+class Ship
+{
+    public function getName()
+    {
+        return 'Starfighter';
+    }
+}
+```
+
+```php
+class JediShip extends Ship
+{
+    public function getFavoriteJedi()
+    {
+        return 'Yoda';
+    }
+}
+```
+
+Suppose we instantiate both objects:
+
+```php
+\$ship = new Ship();
+\$jediShip = new JediShip();
+```
+
+Which of the following lines will cause an error?
+EOF;
+    }
+
+    /**
+     * @param AnswerBuilder $builder
+     */
+    public function configureAnswers(AnswerBuilder $builder)
+    {
+        $builder->addAnswer('`echo $ship->getName();`')
+            ->addAnswer('`echo $ship->getFavoriteJedi();`', true)
+            ->addAnswer('`echo $jediShip->getName();`')
+            ->addAnswer('Both (B) and (C) will cause an error');
```

Comment:
I just realized this won't work, because we don't show the letters on the screen (B) and (C).

Instead, how about we remove the first answer (`echo $ship->getName();`) and then say:

`Both will cause an error`.